vivisect/impemu/emulator.py

```diff
@@ -168,6 +171,9 @@ def checkCall(self, starteip, endeip, op):
             api = self.getCallApi(endeip)
             rtype, rname, convname, callname, funcargs = api
             callconv = self.getCallingConvention(convname)
+            if callconv is None:
+                logger.exception("checkCall(0x%x, 0x%x, %r): cannot get calling convention!", starteip, endeip, op)
```

Comment:
we can force a bail, or we can let the next line cause it for us.  doesn't matter to me.  visi's been big on failing big.  i've always taken the sheepish limp-along way... but this change is about figuring out details about what and where.  otherwise we just get an error message.

Reply:
It's kind of messy and confusing to have logger.exception print the full traceback, and then get it again on line 177. If we're going to limp along, would returning False being better, so that the callers of this don't have to wrap it in the try/catch?

Reply:
returned iscall.